### PR TITLE
Netflix Clone Project Added

### PR DESCRIPTION
**Pull request by hamzathul**
Closes: #833 

- **Title:** Create a Netflix Clone Project 
- **Name:** Hamzathul Favas E
- **Idenitfy yourself:** gssoc-ext, hacktoberfest

### Describe the add-ons or changes you've made 📃
Created a Netflix Clone project in React-Js-Projects -> Intermediate -> Netflix-Clone

### Screenshots 📷

App Screenshot

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: React-JS-Projects/Intermediate/Netflix-Clone/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+</br>
+
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+![App Screenshot](https://github.com/user-attachments/assets/04080ffb-11cc-4ee8-a538-6e6820369b12)
+
+## :zap: Working Video 📹
```
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Intermediate/Netflix-Clone/README.md:69]: Remove this heading, if video is not present.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Add a ss file and review the guidelines properly

File: React-JS-Projects/Intermediate/Netflix-Clone/README.md (hunk 1 above)
Comment (Avdhesh-Varshney) [reply React-JS-Projects/Intermediate/Netflix-Clone/README.md:69]: Keep it under `Basic` folder

Comment (hamzathul) [reply React-JS-Projects/Intermediate/Netflix-Clone/README.md]: Changes have been made

**Review by Avdhesh-Varshney — APPROVED** [review]

**Pull request merged**